assets/queries/cloudFormation/ec2_subnet_unable_connect_cluster/test/positive_expected_result.json

```diff
@@ -0,0 +1,7 @@
+[
+	{
+		"queryName": "EC2 vulnerable public subnet",
+		"severity": "MEDIUM",
+		"line": 7
+	}
+]
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline